demo/index.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="utf8">
+		<script type='module'>
+			import { popup, Menu } from './weasel.js';
+
+			let menu = new Menu([
+				{ name: 'one', action: () => console.warn('one') },
+				{ name: 'two', action: () => console.warn('two') }
+			]);
+
+			function init() {
+				Array.from(document.querySelectorAll('button'))
+				.forEach(el => {
+					el.addEventListener('click', () => triggers[el.id](el));
+				});
+			}
+			document.addEventListener('DOMContentLoaded', init);
+
+			let triggers = [];
```

Comment:
Seems like it should be an object rather than array.

Reply:
🆗